handler/areas.go

```diff
@@ -35,8 +36,18 @@ func NewAreas(cfg *config.Config, r responder, c cantabularClient, d datasetAPIC
 // Get is the handler for GET /population-types/{population-type}/area-types/{area-type}/areas
 func (h *Areas) Get(w http.ResponseWriter, r *http.Request) {
 	ctx := r.Context()
+	limit, _ := strconv.Atoi(r.URL.Query().Get("limit"))
```

Comment:
We shouldn't be letting people give garbage arguments without throwing an error, however I know you plan to improve this across the board so maybe it's good for now